presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java

```diff
@@ -0,0 +1,284 @@
+                " http-server.http.port=" + port + "\n" +
+                " discovery-server.enabled=true\n" +
+                "discovery.uri=http://presto-coordinator:" + port + "\n" +
+                " native-execution-enabled=true\n" +
+                " optimizer.optimize-hash-generation=false\n" +
+                " parse-decimal-literals-as-double=true\n" +
+                " use-alternative-function-signatures=true\n" +
+                " experimental.table-writer-merge-operator-enabled=false\n" +
+                " inline-sql-functions=false\n" +
+                " offset-clause-enabled=true\n" +
+                " regex-library=RE2J\n";
+        Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+        return tempConfigProperties;
+    }
+
+    static Path createCoordinatorJvmProperties()
+    {
+        try {
+            Path tempConfigProperties = Files.createTempFile("jvm", ".properties");
+            String properties = "-server\n" +
+                    "-Xmx1G\n" +
+                    "-XX:+UseG1GC\n" +
+                    "-XX:G1HeapRegionSize=32M\n" +
+                    "-XX:+UseGCOverheadLimit\n" +
+                    "-XX:+ExplicitGCInvokesConcurrent\n" +
+                    "-XX:+HeapDumpOnOutOfMemoryError\n" +
+                    "-XX:+ExitOnOutOfMemoryError\n" +
+                    "-Djdk.attach.allowAttachSelf=true\n";
+            Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+            return tempConfigProperties;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create jvm.properties and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    static Path createCoordinatorLogProperties()
+    {
+        try {
+            Path tempConfigProperties = Files.createTempFile("log", ".properties");
+            String properties = "com.facebook.presto=DEBUG\n";
+            Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+            return tempConfigProperties;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create log.properties and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    public static Path createCoordinatorNodeProperties()
+    {
+        try {
+            Path tempConfigProperties = Files.createTempFile("node", ".properties");
+
+            String properties = "node.environment=testing\n" +
+                    "node.location=testing-location\n" +
+                    "node.data-dir=/var/lib/presto/data\n";
+            Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+
+            return tempConfigProperties;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create config.properties and was exited with error message: " + e.getMessage());
```

Comment:
Just bubble up the exceptions (have methods `throws Exception`).

Reply:
Modified the code accordingly